protect/Makefile

```diff
@@ -0,0 +1,32 @@
+# Definitions
+docker_versions = 1.7.0 1.7.1 1.8.0 1.8.1 1.9.0 1.9.1 1.10.0 1.10.1 1.10.3
```

Comment:
Check out the Docker version range in https://github.com/BD2KGenomics/cgl-docker-lib/pull/214 and use those. Those are the docker versions that actually change the docker protocol version, which is what we care about.